### PR TITLE
feat: Add more Alias actions to AliasContent Manager versioning actions

### PR DESCRIPTION
**Pull request by NarenderRajuB**
Add Below Alias action to AliasContent Manager versioning actions
1.View Usage
2. Change Site and category
3. Rename Alias

### CODE REVIEW

File: djangocms_alias/admin.py [hunk 1]
```diff
@@ -133,8 +135,47 @@ def get_list_actions(self):
...
+        url = admin_reverse('{}_{}_change'.format(
+            obj._meta.app_label, obj._meta.model_name), args=(obj.pk,)
+        )
+        return render_to_string(
+            "admin/djangocms_alias/icons/rename_alias.html",
+            {"url": url, "disabled": False},
+        )
+
+    def _get_alias_usage_link(self, obj, request):
+        """
```
Comment (Aiky30) [inline djangocms_alias/admin.py:161]:
```suggestion
```

File: djangocms_alias/admin.py [hunk 2]
```diff
@@ -133,8 +135,47 @@ def get_list_actions(self):
...
+        Get Alias usage link
```
Comment (Aiky30) [inline djangocms_alias/admin.py:162]:
```suggestion
```

Doesn't add any value.

File: djangocms_alias/admin.py [hunk 3]
```diff
@@ -133,8 +135,47 @@ def get_list_actions(self):
...
+        """
```
Comment (Aiky30) [inline djangocms_alias/admin.py:163]:
```suggestion
```

**Review by Aiky30 — CHANGES_REQUESTED** [review]
Minor changes to the comments mate, we should only add them if they add value.

**Review by Aiky30 — APPROVED** [review]

**Review by adam-murray — APPROVED** [review]

**Pull request merged**